.github/workflows/tests.yaml

```
@@ -19,9 +19,10 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -e .[testing]
-    # CURRENTLY NOT TESTING because it throws an API token error
-    # - name: Run tests
-    #   run: pytest
+    - name: Run tests
+      env:
+          GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
        GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

My eyes have undergone thorough training spotting YAML indentation issues ;) Wouldn't break anything in this case but for good measure?